gradle/plugins/src/main/kotlin/com.hedera.gradle.java.gradle.kts

```diff
@@ -183,6 +183,8 @@ tasks.withType<Javadoc>().configureEach {
             "implSpec:a:Implementation Requirements:",
             "implNote:a:Implementation Note:"
         )
+        options.windowTitle = "Hedera Hashgraph Java SDK"
```

Comment:
this project is not a SDK, its the consensus node or similar

Reply:
What name do you suggest?

Reply:
Changed to 'Hedera Consensus Node'